tests/unit/DatabaseTest.php

```diff
@@ -13,23 +13,30 @@
  */
 class DatabaseTest extends \Codeception\Test\Unit
 {
+    /** @var array Should contain created tables names */
+    public $tableNames = [];
+
+    /** @var string  contain a table name which should create test method*/
+    public $tableName = '' ;
+
     public function _before()
     {
         $config = new DatabaseConfig(
             Directory::fromStringPath(__DIR__ . '/../_data/test-db')
         );
         $this->database = new Database($config);
+        $this->generateRandomTableName();
     }
 
-    public function _after()
+    public function _passed()
     {
         $this->database->delete();
     }
     
     public function testCreateTable(): void
     {
         $table = $this->database->createTable(
-            new TableSchema('test-create-table', [
+            new TableSchema($this->tableName, [
                 new Column('id', ColumnType::STRING(), true),
```

Comment:
Could you move this in a function like so (pseudo-code):

```php
$table = $this->createTestTable('test-create-table');
```
The method could have the following signature:

```
private function createTestTable(string $tableName);
```